### DISCUSSION TITLE
Not clearly about Frame PTS

### DISCUSSION THREAD

**Discussion by luonghoai** [kokorin/Jaffree#326] [Q&A] [answered]
hi all,
Can somebody explain about this comment to me? 
Why Track's timebase is 44100, then we receive timecode is 4410? and How can I define it with milliseconds?
Tks!
![image](https://user-images.githubusercontent.com/43942398/202088621-f2ab2398-aaa7-4c76-889d-7b4d0835609c.png)

**Comment by kokorin** [comment, marked as answer]
`0.1 == 1/10 == 4410 / 44100`

[Let me google for you](https://stackoverflow.com/questions/43333542/what-is-video-timescale-timebase-or-timestamp-in-ffmpeg)
